whirlpool/appliance.py

```diff
@@ -86,12 +86,15 @@ async def fetch_data(self):
             return False
 
         uri = f"{self._backend_selector.base_url}/api/v1/appliance/{self._said}"
-        self._data_dict = None
         async with async_timeout.timeout(30):
-            async with self._session.get(uri) as r:
+            async with self._session.get(uri, headers=self._create_headers()) as r:
                 self._data_dict = json.loads(await r.text())
                 if r.status == 200:
                     return True
+                elif r.status == 401:
+                    await self._auth.do_auth()
+                    await self.start_http_session()
+
```

Comment:
Now that the headers are in PUT/GET maybe we don't need to restart http session here?

Reply:
yeah, we can remove that call